pinot-common/src/main/java/org/apache/pinot/common/metrics/MinionMeter.java

```diff
@@ -31,7 +31,13 @@ public enum MinionMeter implements AbstractMetrics.Meter {
   NUMBER_TASKS_FAILED("tasks", false),
   NUMBER_TASKS_FATAL_FAILED("tasks", false),
   SEGMENT_UPLOAD_FAIL_COUNT("segments", false),
-  SEGMENT_DOWNLOAD_FAIL_COUNT("segments", false);
+  SEGMENT_DOWNLOAD_FAIL_COUNT("segments", false),
+  SEGMENTS_DOWNLOADED("segments", false),
```

Comment:
Can we try to follow one convention for the metric name?

Reply:
Renamed.